### PR TITLE
build: enable release-please

### PR DESCRIPTION
**Pull request by chingor13**
On-boards this repository onto release-please.

An example release pull request: https://github.com/chingor13/functions-framework-dotnet/pull/7

Towards https://github.com/googleapis/repo-automation-bots/issues/3280

### CODE REVIEW

**Comment by jskeet** [comment]
I notice the PR has reformatted all the XML project files, removing the space between the final attribute value and the `/>` in each element. How feasible would it be to not do that? While it's not too much of an issue for this specific repo, it could become much more of a problem in other repositories where the files are generated/edited by different tooling... or would we avoid release-please modifying those files, and instead just modify their source of truth?

File: release-please-config.json [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        {
+          "type": "xml",
+          "path": "/src/CommonProperties.xml",
+          "xpath": "//Project/PropertyGroup/Version"
+        }
+      ]
+    },
+    "src/Google.Cloud.Functions.Hosting": {
+      "component": "Google.Cloud.Functions.Hosting",
+      "extra-files": [
```
Comment (jskeet) [inline release-please-config.json:21]: Is there any way of globbing this? I can easily see us adding a new example but forgetting to update the version numbers. (I've already got a shell script to do the update, if release-please can run that as part of the process...) It's at https://github.com/GoogleCloudPlatform/functions-framework-dotnet/blob/main/update-project-references.sh

**Review by jskeet — COMMENTED** [review]
I notice that the release PR has "Miscellaneous Chores" in some histories but not others - is there a way of excluding them from all histories?

File: release-please-config.json (hunk 1 above)
Comment (chingor13) [reply release-please-config.json:21]: There's currently not a way to glob extra files, but I opened https://github.com/googleapis/release-please/issues/1619 to investigate

**Comment by jskeet** [comment]
> We are using the @xmldom/xmldom library to parse/update the XML document, I'll investigate options on the trailing space, but I'm not optimistic.

Understood. 

Modifying the source of truth: yup. Happy to explore options there.

Miscellaneous chores: I hadn't spotted that that was all from a single synthetic commit. I guess if we have an extra script to fix up formatting etc, it could always remove those sections if that's the simplest way of getting to the right output :)

**Comment by chingor13** [comment]
Closing in favor of #268

**Pull request closed without merging**